std/net/curl.d

```diff
@@ -627,6 +627,61 @@ unittest
     assert(res == cast(ubyte[])[17, 27, 35, 41]);
 }
 
+/* Encode associative array using www-form-urlencoding
+ * perhaps this should live in std.uri
+ */
+private string urlencode()(in string[string] values)
```

Comment:
Even if it's private you should follow phobos symbol naming style, so `urlEncode`.